source/wp-content/themes/wporg-documentation-2022/parts/header.html

```diff
@@ -1,23 +1,23 @@
 <!-- wp:wporg/global-header /-->
 
-<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0"}},"backgroundColor":"charcoal-2","className":"is-sticky","layout":{"type":"constrained"}} -->
+<!-- wp:group {"align":"full","backgroundColor":"charcoal-2","className":"is-sticky","layout":{"type":"constrained"}} -->
 <div class="wp-block-group alignfull is-sticky has-charcoal-2-background-color has-background">
```

Comment:
Do you still need this container group?

Reply:
I thought I would for the semi transparent border to show, but you're right, not required 👍